packages/mobile/src/tokens/saga.ts

```diff
@@ -73,7 +73,14 @@ export function tokenFetchFactory({ actionName, token, actionCreator, tag }: Tok
       const tokenContract = yield call(getTokenContract, token)
       const balanceInWei: BigNumber = yield call([tokenContract, tokenContract.balanceOf], account)
       const balance: BigNumber = yield call(convertFromContractDecimals, balanceInWei, token)
-      CeloAnalytics.track(CustomEventNames.fetch_balance)
+      CeloAnalytics.track(
+        CustomEventNames.fetch_balance,
+        token === CURRENCY_ENUM.DOLLAR
+          ? {
+              dollarBalance: balance,
```

Comment:
balance is a big number, i think we want a string or number representation?

Reply:
Also it might be nice to just have a balance field and then a separate currency field. Makes empty columns less prevalent